testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/exportimport/ExportImportTest.java

```diff
@@ -452,13 +454,6 @@ private void testRealmExportImport() throws LifecycleException {
         // Ensure data are imported back, but just for "test" realm
         Assert.assertNames(adminClient.realms().findAll(), "master", "test", TEST_REALM);
 
-        assertAuthenticated("test", "test-user@localhost", "password");
```

Comment:
@pedroigor I think we need a discussion on this.

When secrets are stripped when exporting e.g. e realm it becomes impossible to migrate a realm by export/import. Not sure whether this is intended.

Reply:
Maybe we should split the PR into use cases where we are sure that secrets should be hidden (e.g. Admin Events) and use cases we need to discuss (e.g. Realm export)